airbyte-integrations/connectors/source-bamboo-hr/README.md

```diff
@@ -70,17 +45,28 @@ docker run --rm -v $(pwd)/secrets:/secrets airbyte/source-bamboo-hr:dev check --
 docker run --rm -v $(pwd)/secrets:/secrets airbyte/source-bamboo-hr:dev discover --config /secrets/config.json
 docker run --rm -v $(pwd)/secrets:/secrets -v $(pwd)/integration_tests:/integration_tests airbyte/source-bamboo-hr:dev read --config /secrets/config.json --catalog /integration_tests/configured_catalog.json
 ```
-
 ## Testing
-You can run our full test suite locally using [`airbyte-ci`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md):
-```bash
-airbyte-ci connectors --name=source-bamboo-hr test
-```
 
-### Customizing acceptance Tests
+#### Acceptance Tests
 Customize `acceptance-test-config.yml` file to configure tests. See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference) for more information.
 If your connector requires to create or destroy resources for use during acceptance tests create fixtures for it and place them inside integration_tests/acceptance.py.
 
+To run your integration tests with Docker, run:
+```
+./acceptance-test-docker.sh
```

Comment:
This is not correct. We've fully switched to using `airbyte-ci connectors --name source-bamboo-hr test` for this.